dist/edu-benefits-schema.json

```diff
@@ -662,7 +606,7 @@
             "$ref": "#/definitions/dateRange"
           },
           "hours": {
-            "type": "integer"
+            "type": "number"
```

Comment:
@jbalboni I thought the outcome for this was to prevent floats on the front-end, not allow them in the schema?